app/javascript/react/containers/AirlineShowContainer.js

```diff
@@ -35,6 +36,7 @@ class AirlineShowContainer extends Component {
           destinations={this.state.destinations}
           rating={this.state.rating}
         />
+        <ReviewIndexContainer />
       </div>
     )
```

Comment:
It would be great if this show component also held `reviews` state and passed it down to the ReviewIndexContainer